config/jobs/periodic/kubeflow/kubeflow-e2e-periodics.yaml

```diff
@@ -0,0 +1,87 @@
+periodics:
+  - name: periodic-kubeflow-pipeline-e2e-test-ppc64le
+    cluster: kubeflow-ppc64le-cluster
+    labels:
+      preset-kubeflow-powervs: "true"
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: ppc64le-kubernetes
+        path_strategy: explicit
+      gcs_credentials_secret: gcs-credentials
```

Comment:
You would need the `gcs-credentials` secret to be present in your cluster to push the job artifacts to our GCS bucket.

Instead, I suggest you create an IBM S3 COS bucket in your account and create s3 credential secret in your ppc build cluster to push logs to that s3 bucket.
The bucket would have service-account.json and the secret manifest can be in below format:
```
---
apiVersion: v1
kind: Secret
metadata:
  namespace: prow
  name: kubeflow-s3-credentials
stringData:
  service-account.json: |
    {
      "region": "us-south",
      "access_key": "xxxx",
      "endpoint": "s3.us-south.cloud-object-storage.appdomain.cloud",
      "insecure": false,
      "s3_force_path_style": true,
      "secret_key": "xxxx"
    }
```

The metadata here would be like https://github.com/ppc64le-cloud/test-infra/blob/master/config/prow/config.yaml#L77:L80

Reply:
Have added the s3 credential secret in our ppc build cluster, and made the necessary config changes